### PR TITLE
tester_env

### PR DESCRIPTION
**Pull request by nyodas**
@puckel 
Fix issue when we want to add environment variable to the tester only.

### CODE REVIEW

File: dgr/aci-test.go [hunk 1]
```diff
@@ -140,6 +141,11 @@ func (aci *Aci) buildTestAci() (string, error) {
 		isolators = aci.manifest.Aci.App.Isolators
 	}
 
+	var environment []types.EnvironmentVariable
+	if tmpEnv := append(aci.manifest.Tester.Aci.App.Environment, aci.manifest.Aci.App.Environment...); len(tmpEnv) != 0 {
```
Comment (n0rad) [inline dgr/aci-test.go]: I don't like the append on if, please split in 2

**Comment by nyodas** [comment]
Fixed the comment ,splited the if.

File: dgr/aci-test.go (hunk 1 above)
Comment (n0rad) [inline dgr/aci-test.go]: this is wrong, append should end up in same array

File: dgr/aci-test.go [hunk 2]
```diff
@@ -140,6 +141,11 @@ func (aci *Aci) buildTestAci() (string, error) {
...
+		environment = underscore.UniqBy(tmpEnv, "Name").([]types.EnvironmentVariable)
```
Comment (n0rad) [inline dgr/aci-test.go]: you don't need to change 456 files to do a uniq

Comment (nyodas) [inline dgr/aci-test.go]: Vendor install + glide madness

File: dgr/aci-test.go (hunk 1 above)
Comment (nyodas) [inline dgr/aci-test.go]: Fixed
even if i don't particularly agree w/h you on the same array stuff.

**Comment by n0rad** [comment]
I think it's still wrong, if there is no test vars, you do not push the app vars and push only an empty list

**Comment by nyodas** [comment]
I don't get it.
By default aci.manifest.Tester.Aci.App.Environment and aci.manifest.Aci.App.Environment
Are typed with `[]types.EnvironmentVariable`
Both are initialized as empty slice.
Therefore the appending of an empty slice on an empty slice shouldn't be problematic.
In the end there wasn't any env variable set up in the manifest , it stand to reason that there is none in the resulting aci.
Additionally if you append stuff to an empty slice , the slice grows with the new stuff. 
I added a few more bats tests to confirm this.

**Comment by n0rad** [comment]
My bad I misread the change

**Pull request merged**